<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EbsdImportWidget</class>
 <widget class="QWidget" name="EbsdImportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="Line" name="line_8">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QGroupBox" name="originGroupBox">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
        </property>
        <property name="title">
         <string>Reference Frame Options</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <property name="horizontalSpacing">
          <number>4</number>
         </property>
         <property name="verticalSpacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="m_RefFrameOptionsBtn">
           <property name="toolTip">
            <string>This will show the advanced reference frame options</string>
           </property>
           <property name="text">
            <string>Set Reference Frame</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="stackingGroupBox">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>This is the order that the input files are arranged in XYZ space so that a specific file is equivelent to the physical Z=0 Origin</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
        </property>
        <property name="title">
         <string>Stacking Order</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="m_StackLowToHigh">
           <property name="text">
            <string>Low To High</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="m_StackHighToLow">
           <property name="text">
            <string>High To Low</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="zspacingGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
        </property>
        <property name="title">
         <string>Z Spacing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="m_zSpacing">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>The spacing between each layer.</string>
           </property>
           <property name="text">
            <string>0.25</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_53">
       <property name="styleSheet">
        <string>QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Output File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFSDropLineEdit" name="m_OutputFile">
       <property name="toolTip">
        <string>The absolute path to the output file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_OutputFileBtn">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line_9">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_58">
     <property name="styleSheet">
      <string notr="true">QLabel 
{
	font-style: italic;font-weight: bold;
font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string>Advanced File Name Options (You generally should NOT have to change these values)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QGridLayout" name="gridLayout_20">
     <item row="1" column="0">
      <widget class="QLabel" name="label_50">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Start Slice:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="m_ZStartIndex">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Which file index to start the import at.</string>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_51">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>End Slice:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="m_ZEndIndex">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The file index to stop the import at. (Inclusive)</string>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_21">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Padding Digits:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QSpinBox" name="m_TotalDigits">
       <property name="toolTip">
        <string>The number of digits to allow for the slice index.</string>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_32">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>File Prefix:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_FilePrefix">
       <property name="toolTip">
        <string>The front part of the input file name</string>
       </property>
       <property name="text">
        <string>Slice_</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>File Suffix:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="m_FileSuffix">
       <property name="toolTip">
        <string>The part of the filename that comes AFTER the slice index</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_56">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>File Extension:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="m_FileExt">
       <property name="toolTip">
        <string>The file extension of the input files</string>
       </property>
       <property name="text">
        <string>ang</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="_6">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
       </property>
       <property name="text">
        <string>Generated Input Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_GeneratedFileNameExample">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>[Prefix][Zero Padding][Slice Value][Suffix].[extension]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_25">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_57">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>File List (Green=File Exists  Red=File Does NOT Exist)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QListWidget" name="m_FileListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The list of files that will be imported in the order they will be imported into the HDF5 Ebsd File</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="m_ErrorMessage">
     <property name="text">
      <string>Error Messages</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QProgressBar" name="m_progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_GoBtn">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox 
{
	font-style: italic;font-weight: bold;
  font-size: 12px;
  }
</string>
     </property>
     <property name="title">
      <string>Source Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_29">
          <property name="toolTip">
           <string>The directory that contains the .ang or .ctf files</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>Input Directory:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFSDropLineEdit" name="m_InputDir"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="m_InputDirBtn">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_47">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-weight: bold;
font-size: 11px;
}</string>
          </property>
          <property name="text">
           <string>Total Ebsd Files Found:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="m_TotalSlices">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFSDropLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QFSDropLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
